hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Modal from "@material-ui/core/Modal";
+import Button from "@material-ui/core/Button";
+import styles from "components/general/PopupModal/PopupModal.module.scss";
+import { Box } from "@material-ui/core";
+
+interface PopupModalProps {
+    title?: string;
+    description: string;
+    isVisible: boolean;
+    submitText?: string;
+    cancelText?: string;
+    submitHandler(): any;
+    cancelHandler?(): any;
+}
+
+const PopupModal = ({
+    title,
+    description,
+    isVisible,
+    submitText,
+    cancelText,
+    submitHandler,
+    cancelHandler,
+}: PopupModalProps) => {
+    return (
+        <Modal open={isVisible}>
+            <Box className={styles.modalSize}>
+                <h1 className={styles.title}>{title}</h1>
```

Comment:
display title conditionally because it is a conditional prop e.g. `{title && <h1..../>}`

Reply:
yes